Initialization of OracleEnvironment fails to initialize if Oracle 12c JDBC driver is used

Initialization of OracleEnvironment fails to initialize if Oracle 12c JDBC driver is used.

```
java.lang.Error: java.lang.reflect.InvocationTargetException
    at dbfit.api.DbEnvironmentFactory$EnvironmentDescriptor.createEnvironmentInstance(DbEnvironmentFactory.java:68)
    at dbfit.api.DbEnvironmentFactory.createEnvironmentInstance(DbEnvironmentFactory.java:102)
    at dbfit.api.DbEnvironmentFactory.newEnvironmentInstance(DbEnvironmentFactory.java:106)
    at dbfit.OracleTest.(OracleTest.java:5)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
    at java.lang.Class.newInstance(Class.java:374)
    at fit.FixtureClass.newInstance(FixtureClass.java:24)
    at fit.FixtureLoader.instantiateFixture(FixtureLoader.java:62)
    at fit.FixtureLoader.instantiateFirstValidFixtureClass(FixtureLoader.java:84)
    at fit.FixtureLoader.disgraceThenLoad(FixtureLoader.java:44)
    at fit.Fixture.loadFixture(Fixture.java:141)
    at fit.Fixture.getLinkedFixtureWithArgs(Fixture.java:133)
    at fit.Fixture.doTables(Fixture.java:78)
    at fit.FitServer.process(FitServer.java:81)
    at fit.FitServer.run(FitServer.java:56)
    at fit.FitServer.main(FitServer.java:41)
Caused by: java.lang.reflect.InvocationTargetException
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
    at dbfit.api.DbEnvironmentFactory$EnvironmentDescriptor.createEnvironmentInstance(DbEnvironmentFactory.java:65)
    ... 18 more
Caused by: java.lang.Error: Cannot initialise oracle rowset
    at dbfit.environment.OracleEnvironment.(OracleEnvironment.java:203)
    ... 23 more
Caused by: java.lang.ClassNotFoundException: oracle.jdbc.driver.OracleResultSetImpl
    at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Class.java:190)
    at dbfit.environment.OracleEnvironment.(OracleEnvironment.java:199)
    ... 23 more
```

I'll check if helps to switch switching the following to some parent interface/class

``` java
            TypeNormaliserFactory.setNormaliser(
                    Class.forName("oracle.jdbc.driver.OracleResultSetImpl"),
                    new OracleRefNormaliser());
```

--- a/dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java
+++ b/dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java
@@ -229,8 +229,7 @@ public class OracleEnvironment extends AbstractDbEnvironment {
         TypeNormaliserFactory.setNormaliser(java.sql.Date.class,
                 new SqlDateNormaliser());
         try {
-            TypeNormaliserFactory.setNormaliser(
-                    Class.forName("oracle.jdbc.driver.OracleResultSetImpl"),
+            TypeNormaliserFactory.setNormaliser(java.sql.ResultSet.class,
                     new OracleRefNormaliser());
         } catch (Exception e) {
             throw new Error("Cannot initialise oracle rowset", e);

